test/integration/gcp/controls/forseti.rb

```diff
@@ -0,0 +1,136 @@
+
+gcp_project_id = attribute('gcp_project_id')
+#gcp_org_id = attribute('gcp_org_id')
+
+control 'forseti-client-vm' do
+  impact 1.0
+  title 'Test forseti client vm'
+  describe google_compute_instances(project: gcp_project_id,  zone: 'us-central1-c') do
+    its('instance_names') { should include /forseti-client-vm-*/ }
+  end
+end
+
+control 'forseti-server-vm' do
+  impact 1.0
+  title 'Test foseti server vm'
+  describe google_compute_instances(project: gcp_project_id,  zone: 'us-central1-c') do
+    its('instance_names') { should include /forseti-server-vm-*/ }
+  end
+end
+
+control 'forseti-cloudsql-instance' do
+  impact 1.0
+  title 'Test CloudSQL Instance is up'
+  describe google_sql_database_instances(project: gcp_project_id) do
+    its('instance_names') { should include /forseti-server-db-*/ }
+  end
+end
+
+
+control 'forseti-server-iam-roles' do
+  gcp_enable_privileged_resources = '1'
+
+  impact 1.0
+  title 'Test Server Project IAM Role bindings'
+  describe google_project_iam_binding(project: gcp_project_id, role: "roles/storage.objectViewer") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: gcp_project_id, role: "roles/storage.objectCreator") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: gcp_project_id, role: "roles/cloudsql.client") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: gcp_project_id, role: "roles/cloudtrace.agent") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: gcp_project_id, role: "roles/logging.logWriter") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+  describe google_project_iam_binding(project: gcp_project_id, role: "roles/iam.serviceAccountTokenCreator") do
+    it { should exist }
+    its ('members'){ should include /forseti/ }
+  end
+end
+
+# @TODO Need to figure out the Org level permissions to execute this test
+#
+# control 'forseti-server-read-iam-roles' do
+#   gcp_enable_privileged_resources = '1'
+#
+#   impact 1.0
+#   title 'Test Server Read Org Level IAM Role bindings'
+#   describe google_project_iam_binding(project: , role: "roles/compute.securityAdmin") do
+#     it { should exist }
+#     its ('members'){ should include /forseti/ }
+#   end
+# end
+
+control 'forseti-google-storage-buckets' do
+  impact 1.0
+  title 'Test GCS Buckets are present'
+  describe google_storage_buckets(project: gcp_project_id) do
+    its('bucket_names') { should include /forseti-server/ }
+  end
+  describe google_storage_buckets(project: gcp_project_id) do
+    its('bucket_names') { should include /forseti-client/ }
+  end
+  # @TODO can't get the bucket to accept regex matching, below works but doesn't account for random_id
```

Comment:
More commented out tests. I can poke at this when I perform FR on this pull request.